cli/determined_cli/experiment.py

```diff
@@ -235,7 +235,7 @@ def _parse_config_file_or_exit(config_file: io.FileIO) -> Dict:
 
 @authentication_required
 def submit_experiment(args: Namespace) -> None:
-    experiment_config = _parse_config_file_or_exit(args.config_file)
+    experiment_config = _parse_config_file_or_exit(args.config_file + "Fake name")
```

Comment:
remove?

Reply:
Wanted to see it run in CI first, will remove after.